packages/cozy-client/src/manifest.js

```diff
@@ -0,0 +1,85 @@
+const AUTHORIZED_CATEGORIES = [
+  'banking',
+  'cozy',
+  'energy',
+  'health',
+  'host_provider',
+  'insurance',
+  'isp',
+  'mes_infos',
+  'online_services',
+  'partners',
+  'press',
+  'productivity',
+  'ptnb',
+  'public_service',
+  'shopping',
+  'social',
+  'telecom',
+  'transport'
+]
+
+/** Filters unauthorized categories. Defaults to ['others'] if no suitable category. */
+export function sanitizeCategories(categories) {
+  if (!categories) return ['others']
+  const filteredList = categories.filter(c => AUTHORIZED_CATEGORIES.includes(c))
+  if (!filteredList.length) return ['others']
+  return filteredList
+}
+
+export function areTermsValid(terms) {
+  return Bolean(terms && terms.id && terms.url && terms.version)
+}
+
+export function isPartnershipValid(partnership) {
+  return Boolean(partnership && partnership.description)
+}
+
+/**
+ * Normalize app manifest, retrocompatibility for old manifests
+ *
+ * @param  {Manifest} manifest
+ * @return {Manifest}
+ */
+export function sanitize(manifest) {
+  const sanitized = { ...manifest }
+
+  // Make categories an array and delete category attribute if it exists
+  if (
+    !manifest.categories &&
```

Comment:
It's probably not for this PR but I really dislike these options that we have to maintain a long time. Should we create an issue about deprecating the  `category` option and only leave `categories`?

Reply:
I do not know the status of the category option, I merely moved the code from the store here.

I have no strong opinion on the deprecation here, I do not know for how long the category option has been removed.